src/core/simulation.jl

```diff
@@ -213,7 +213,7 @@ end
                 )
 
 """
-mutable struct Simulation
+mutable struct Simulation <: IS.InfrastructureSystemsType
```

Comment:
I'm not sure this change is appropriate.  InfrastructureSystemsType is typically a thing you store in a system.  An undesirable side effect is that called `methodswith(Simulation, supertypes=true)` is going to report methods that do not exist for this type.

Reply:
Ok, I will remove it.